pkg/specs/interface.go

```diff
@@ -75,13 +75,25 @@ func (r *Resolver) ResolveRelease(ctx context.Context, upstream string) (*api.Re
 			false,
 		)
 
+	case "runbook.replicated.app":
```

Comment:
what is this?

Reply:
i need to return something different for that switch statement. i'm just following the pattern